src/System/LoadingFailedException.cs

```diff
@@ -1,14 +1,12 @@
 using System;
-using System.Runtime.Serialization;
 
-namespace SFML
+namespace SFML.System
 {
     ////////////////////////////////////////////////////////////
     /// <summary>
     /// Exception thrown by SFML whenever loading a resource fails
     /// </summary>
     ////////////////////////////////////////////////////////////
-    [Serializable]
```

Comment:
It doesn't, but i heard it is going to come back for 2.0 i'll have to check that again